scripts/common.py

```diff
@@ -33,24 +33,28 @@ def execute(cmd, abort = True):
     return (status, stdout.rstrip('\n'), stderr.rstrip('\n'))
 
 
-# Subroutine for writing "pretty" XML; copied from http://effbot.org/zone/element-lib.htm#prettyprint
 def indent(elem, level=0):
-  i = "\n" + level*"  "
-  if len(elem):
-    if not elem.text or not elem.text.strip():
-      elem.text = i + "  "
-    if not elem.tail or not elem.tail.strip():
-      elem.tail = i
-    for elem in elem:
-      indent(elem, level+1)
-    if not elem.tail or not elem.tail.strip():
-      elem.tail = i
-  else:
-    if level and (not elem.tail or not elem.tail.strip()):
-      elem.tail = i
+    """Subroutine for writing "pretty" XML; copied from
+    http://effbot.org/zone/element-lib.htm#prettyprint"""
+    i = "\n" + level*"  "
```

Comment:
Could the two-space string be a module or at least subroutine variable rather than a constant which needs to appear more than once? It could even make the function configurable by adding a new optional argument.

Reply:
See above.